skt/__init__.py

```diff
@@ -466,6 +468,12 @@ def prepare(self, clean=True):
 
         shutil.copyfile(self.basecfg, "%s/.config" % self.path)
 
+        # Disable debuginfo by default unless we have been asked to enable it.
+        if not self.enable_debuginfo:
+            logging.info("disabling debuginfo:")
+            args = shlex.split('script/config --disable debug_info')
+            subprocess.check_call(args)
```

Comment:
This should probably just be:

```
subprocess.check_call(['script/config', '--disable', 'debug_info'])
```

And drop the ```shlex``` usage? What do you think?

Reply:
I'm not opposed, but `shlex` is usually my go-to for increasing readability. ;)

I'll change it.